### ISSUE TITLE
TypeError - Plugin Breaks System

### ISSUE THREAD

**Issue by DrewBold** [barrelstrength/craft-sprout-seo#263]
### Steps to reproduce

1. Use {% sproutseo 'optimize' %}
2. Error

### Additional info

- Craft version: Craft Pro 3.7.53.1
- PHP version: 7.4.19
- Database driver & version:   MySQL 5.7.34
- Plugins & versions:Sprout SEO PRO 4.6.9

<img width="1703" alt="Screenshot 2022-09-13 at 10 45 10 AM" src="https://user-images.githubusercontent.com/66840687/189932604-b056f70e-299d-4c47-8973-dd8765e48c0c.png">
<img width="1699" alt="Screenshot 2022-09-13 at 10 45 37 AM" src="https://user-images.githubusercontent.com/66840687/189932631-c2b63104-a9df-48e0-a52b-29ad7520b30f.png">

<img src="https://front.com/assets/img/favicons/favicon-32x32.png" height="16" width="16" alt="Front logo" /> [Front conversations](https://app.frontapp.com/open/top_4p7gt)

**Comment by alexagui** [comment]
I'm having the same issue.

- Craft Pro 3.7.54
- PHP 8.0.23
- MariaDB 10.3.36
- Sprout SEO 4.6.9

**Comment by lexislav** [comment]
I have the same issue here. It happens after Formie plugin update. At least in my case. Only happens on production server. I am not able to recreate on localhost.

PHP version | 8.1.8
MySQL 10.1.48
Imagick 3.7.0 (ImageMagick 6.9.7-4)
Craft Pro 3.7.55.3
Sprout SEO 4.6.9 (there is no other version for long time AFAIK)

**Comment by CelularPrism** [comment]
I have the same problem when logging into Sprout SEO

PHP version: 7.4.29
MySQL: 5.5.5
Imagick: 3.7.0
Craft Pro: 3.7.46
Sprout SEO: 4.6.9

**Comment by rjanits** [comment]
I have the same problem as well.

PHP version: 7.4.32
MySQL: 5.7
Craft CMS (Pro): 3.7.56
Sprout SEO (Pro): 4.6.9

**Comment by alexagui** [comment]
A quick & dirty fix is to update line 130 in `vendor/barrelstrength/sprout-base-fields/src/models/Address.php` to

```
$subdivision = $subdivisionRepository->get($this->administrativeAreaCode || '', [$this->countryCode]);
```

This is obviously not ideal but will get your site running again. 

![Image 2022-10-26 at 10 46 11 PM](https://user-images.githubusercontent.com/685/198178880-4a19d028-acf6-419c-979c-f8fd53277c47.jpg)

**Comment by lexislav** [comment]
Thanks for this!
I have already lost my believe, that barrelstrenght will update any of his plugins.
So until I will switch to another solution, this is almost perfect!

**Comment by DrewBold** [comment]
We have already abandoned Sprout and have switched to another provider for all of our projects and clients. Taking 1.5 months to respond and address this issue is not acceptable - especially when it breaks the system. 

We not only reached out here but sent several emails directly to Sprout with no reply. 

For a paid plugin/service we cannot in good faith recommend or deploy your products or services any longer.

**Comment by BenParizek** [comment]
I'm adding this bug to the top of my list to review and will try to have a fix out soon.

The [Road Map](https://sprout.barrelstrengthdesign.com/docs/support/road-map.html) (recently updated with a few more comments) continues to be the best summary of where things stand as I continue on the long road to catching up and getting back to a more responsive level of support.

**Comment by BenParizek** [comment]
A fix for this has been released in Sprout SEO v4.6.10
